Rubberduck.Core/UI/CodeExplorer/Commands/CodeExplorerCommandBase.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using Rubberduck.UI.Command;
+using Rubberduck.UI.Command.ComCommands;
+using Rubberduck.VBEditor.Events;
 
 namespace Rubberduck.UI.CodeExplorer.Commands
 {
-    public abstract class CodeExplorerCommandBase : CommandBase
+    public abstract class CodeExplorerCommandBase : ComCommandBase
```

Comment:
If I understand correctly, the `ExportCommand` in the referenced PR still does COM access, so it's still appropriate that it's derived from `CodeExplorerCommandBase` command. Note that "COM access" can be quite broad - simply taking a reference to a `IVBE` or the project repository is IMO a COM access and must be managed accordingly. 

Also, IIRC, I decided that all CE commands should be `ComCommand` because CE is a UI tool and it doesn't make sense to use it if everything is going down anyway.